modules/dsfhub-azure-sql-managed-instance/main.tf

```diff
@@ -0,0 +1,36 @@
+terraform {
+  required_providers {
+    dsfhub = {
+      source = "imperva/dsfhub"
+    }
+  }
+}
+
+resource "dsfhub_data_source" "this" {
+  server_type = "AZURE SQL MANAGED INSTANCE"
```

Comment:
did you happen to test whether `Service Name` is required for this source?

Reply:
No, but by virtue of omission I can say it ain't required (it's not used for connecting anywhere either)